src/combinatorics/simple_graph/basic.lean

```diff
@@ -233,7 +233,7 @@ instance : boolean_algebra (simple_graph V) :=
 
 @[simp] lemma empty_graph_eq_bot (V : Type u) : empty_graph V = ⊥ := rfl
 
-instance (V : Type u) : inhabited (simple_graph V) := ⟨⊤⟩
+@[simps] instance (V : Type u) : inhabited (simple_graph V) := ⟨⊥⟩
```

Comment:
Does this generate a reasonably named lemma? In my experience `@[simps]` does a bad job on instances.

Reply:
It gives `simple_graph.inhabited_default`

Reply:
Yeah, that's not great; in general, we don't want the typeclass name in projections, because typically we export the projection without that prefix.

I'd prefer `default_def` or `default_eq` after the dot, but I guess it's not really worth worrying about